test/src/main/java/org/apache/accumulo/test/functional/ManagerAssignmentIT.java

```diff
@@ -413,24 +415,41 @@ public void testOpidPreventsAssignment() throws Exception {
         writer.addMutation(m);
       }
 
+      // Host all tablets.
       c.tableOperations().setTabletHostingGoal(tableName, new Range(), TabletHostingGoal.ALWAYS);
+      Wait.waitFor(() -> countTabletsWithLocation(c, tableId) == 3);
```

Comment:
Could make the test check the tablet after waiting to ensure the expected tablet does not have a location

```suggestion
      Wait.waitFor(() -> countTabletsWithLocation(c, tableId) == 3);
      var ample = ((ClientContext)c).getAmple();
      assertNull(ample.readTablet(new KeyExtent(tableId, new Text("m"), new Text("f"))).getLocation());
```

Reply:
Added in 6ab6766